core/src/main/java/io/ably/lib/transport/ITransport.java

```diff
@@ -73,13 +72,11 @@ public Param[] getConnectParams(Param[] baseParams) {
                 paramList.add(new Param("resume", connectionKey));
                 if(connectionSerial != null)
```

Comment:
Can `connectionSerial` be removed now?

Reply:
> Just to check - looking at the presence tests in `RealtimePresenceTest.java` - the majority of the tests have an `@Ignore` annotation? Does this mean they could be failing?
> 
> edit: the recovery tests also have `@Ignore` annotations?

I have raised an issue about the ignored tests in the java channel. We will need to see what to do with those.

Reply:
Ok - though its not clear how the presence changes are being tested?

Reply:
This part is removed as it is not used at all.